cime_config/usermods_dirs/NEON/FATES/STER/shell_commands

```diff
@@ -0,0 +1,7 @@
+./xmlchange NEONSITE=STER
+./xmlchange PTS_LON=256.96992
+./xmlchange PTS_LAT=40.45984
+# Setup to run with prognostic crops for this site
+# If you want to explicitly run in SP mode or add other
+# # options you'll need to add that after this...
+./xmlchange CLM_BLDNML_OPTS="--bgc bgc --crop"
```

Comment:
Hmmm yeah I'm not sure what to do about these crop sites...

Reply:
The three options that I see would be to:
- Delete the crop site for FATES, so it wouldn't be an option
- Run it without crop for FATES, but maybe only allow a single grassland PFT? So the configuration would be a little different than other sites.
- Keep the directory, but set it up so that it fails with a warning that this is a crop site and you can't run FATES that way

I think it would be OK to delete the crop sites for FATES since you probably do want to run the full prognostic crop model for these sites, and you would do that from the big-leaf version. As such possibly that last option where it fails with an error is the best.

Reply:
> I think it would be OK to delete the crop sites for FATES since you probably do want to run the full prognostic crop model for these sites, and you would do that from the big-leaf version. As such possibly that last option where it fails with an error is the best

Okay that makes sense to me. How should we fail the run?

Reply:
We can add an exit statement to the bash shell_commands script.